tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -166,7 +166,7 @@ def test_tuples(cartesian_case):  # noqa: F811 # fixtures
     @gtx.field_operator
     def testee(a: cases.IJKFloatField, b: cases.IJKFloatField) -> cases.IJKFloatField:
         inps = a, b
-        scalars = 1.3, float64(5.0), float64("3.4")
+        scalars = 1.3, float64(5.0), float64(3.4)
```

Comment:
This change is ok, here, but we need the functionality in general (and hopefully there is another test covering this). The use-case is `float32("0.1")` which means take the closest float32 of the string literal, while `float32(0.1)` would create a float64 literal and then cast to float32, which is different (if not in this case, then you can google for the sequence where it matters).

Reply:
It's possible to add a `StringType`, but since this is the one and only case where it would be used, it may be better to address this in the lowering so as to not litter the type system and mislead people into thinking there are actual strings in the DSL. Agreed, adding a `StringType` is probably cleaner in code.

If I understand correctly, this difference shows up for less than 0.000000186264514923% of the numbers, and all it does is it flips the direction in which a 0.5 ULP error is introduced, which won't make any difference in the overwhelming majority of cases.

The return on investment on this feature is just so low that I don't see it worth to spend time on it. IMO we should focus on much more pressing issues, like the design of the core IRs.